VimeoNetworking/Sources/Models/Spatial.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Spatial.swift
+//  Pods
+//
+//  Created by Lim, Jennifer on 1/6/17.
+//
+//
+//
+
+/// Spatial stores all information related to threesixty video
+public class Spatial: VIMModelObject
+{
+    
+    /// Represents the projection. Value returned by the server can be: "equirectangular", "cylindrical", "cubical", "pyramid", "dome".
+    dynamic var projection: String?
```

Comment:
I guess if we ever need to take advantage of KVO for these then we can change it back. In the meantime we'll enjoy that sweet, sweet minuscule and imperceptible performance improvement by not making it dynamic! Woooo 🏃🏻‍♀️💨